Tests/StreamChatUITests/SnapshotTests/ChatThread/ChatThreadVC_Tests.swift

```diff
@@ -106,6 +106,34 @@ final class ChatThreadVC_Tests: XCTestCase {
         )
     }
 
+    func test_whenThreadRendersParentMessageEnabledIsFalse() {
```

Comment:
I've opted to follow the same pattern in the rest of the tests in ChatThreadVC_Tests, for example: `test_whenShouldMessagesStartAtTheTopIsTrue()`

When the expected result is too obvious, I don't mind omitting it, but I don't mind being explicit, either. I think, in the end, it is just personal taste. 

In this case, because it is consistent with the rest of the tests in this class, I'm willing not to change it, but let me know if you have a different opinion 👍